### ISSUE TITLE
can not publish command.

### ISSUE THREAD

**Issue by satoshikumano** [KiiPlatform/gateway-agent#130]
Failed to publish command.

gateway-agent: 3d8a111e21698c4efa2d24b42ae6f5340557675a (improve_onboard)
gwm-cli: 38ff1ed2596f6d208acebbdf14b651f8e82847cb (master)
gwc-cli: 1521576d539c3ca857278d620c97ade49d94e044 (imprv)

Steps:
### gwm-cli:
```shell
./gwm-cli auth --username admin_user --password admin_pass --app-name master
./gwm-cli onboard-gateway --app-name master
./gwm-cli user-login --username ${kiiuser name} --password ${kii user password} --app-name master
./gwm-cli add-owner --gateway-password ${gw password} --app-name master
```

### gwc-cli
```shell
./gwc-cli -evid node1 -aname master
1
```

### gwm-cli:
```
./gwm-cli onboard-node --node-vid node1 --node-password {end-node password} --app-name master
./gwm-cli s -node-vid node1 --app-name master
```

**Comment by yongpingchen** [comment]
caused by gateway-agent doesn't update converter-endnodes mapping after it is notified with new onboarded end-node thingID

**Comment by yongpingchen** [comment]
caused by 2 reasons:
1. commandTopic for endnode address is wrong(fixed on #131 )
2. new endnode for converters is added after converter reported connection of endnode

**Comment by satoshikumano** [comment]
Fixed.
